pkg/util/pravegacluster.go

```diff
@@ -79,3 +81,17 @@ func LabelsForPravegaCluster(pravegaCluster *v1alpha1.PravegaCluster, component
 func PravegaControllerServiceURL(pravegaCluster v1alpha1.PravegaCluster) string {
 	return fmt.Sprintf("tcp://%v.%v:%v", ServiceNameForController(pravegaCluster.Name), pravegaCluster.Namespace, "9090")
 }
+
+func PvcIsOrphan(stsPvcName string, replicas int32) bool {
+	index := strings.LastIndexAny(stsPvcName, "-")
+	if index == -1 {
+		return false
+	}
+
+	ordinal, err := strconv.Atoi(stsPvcName[index+1:])
+	if err != nil {
+		return false
+	}
+
+	return int32(ordinal) >= replicas
```

Comment:
I think the ordinal can be zero and replica is at least one. I'm not sure this logic is correct.

Reply:
Not quite sure what the problem is.
Ordinal being zero means the current number of pvc is 1. And replicas being 1 means the current target number of pvc is 1, which is just fine in this case. So 0 >= 1 is false and this pvc is not orphan. 
If ordinal is 1 and replicas is 1, then we have pvc number being 2 and target number of pvc being 1. The pvc with ordinal 1 shoud be deleted. So 1 >= 1 will return true. 
BTW ordinal is started from 0.